tests/integration/.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
+*_log.txt
```

Comment:
Please add a newline at the end of this line

Reply:
ok